src/components/Card.js

```diff
@@ -1,52 +1,85 @@
 export default class Card {
-  constructor({ name, link }, cardSelector, handleImageClick) {
+  constructor(
+    { name, link, _id, isLiked },
+    cardTemplate,
+    handleImageClick,
+    handleTrashClick,
+    api
+  ) {
     this._name = name;
     this._link = link;
-    this._cardSelector = cardSelector;
+    this._id = _id;
+    this.isLiked = isLiked;
+    this._cardTemplate = cardTemplate;
     this._handleImageClick = handleImageClick;
-    this._element = this._getTemplate();
-    this._likeButton = this._element.querySelector(".card__like-button");
-    this._deleteButton = this._element.querySelector(".card__delete-button");
-    this._cardImage = this._element.querySelector(".card__image");
-    this._cardTitle = this._element.querySelector(".card__title");
+    this._handleTrashClick = handleTrashClick;
+    this._api = api;
+    this.element = this._getTemplate();
+    this.likeButton = this.element.querySelector(".card__like-button");
+    this._deleteButton = this.element.querySelector(".card__delete-button");
+    this._cardImage = this.element.querySelector(".card__image");
+    this._cardTitle = this.element.querySelector(".card__title");
+    this._deleteCardModal = document.querySelector("#delete-card-modal");
```

Comment:
`    this._deleteCardModal = document.querySelector("#delete-card-modal");
`

It's not needed because `_handleTrashClick` is not needed in line 67